Src/Enums/theme_elements.py

```
@@ -0,0 +1,190 @@
+from enum import Enum
+
+class ThemeElement(Enum): pass
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)